readers/elasticsearch/src/test/java/com/criteo/hadoop/garmadon/elasticsearch/UriHelperTest.java

```diff
@@ -1,5 +1,6 @@
 package com.criteo.hadoop.garmadon.elasticsearch;
```

Comment:
This file should be moved to readers common tests

Reply:
done